src/Burger.tsx

```diff
@@ -37,6 +38,6 @@ export const Layer = ({ type }) => {
 // As we're building from the bottom up, reverse the layer order
 export const Burger = ({ layers = [] }) => (
   <Flex justifyContent="center" alignItems="center" position={[-0.5, 0, 0]}>
-    {layers.map((type, key) => <Layer key={key} type={type} />).reverse()}
+    {layers.map((type, key) => <Layer key={key} type={type} />)}
```

Comment:
Are you reversing the array in the editor instead?

Reply:
yeah